R/DittoScatterPlot.R

```diff
@@ -1,5 +1,6 @@
 #' Show RNAseq data overlayed on a scatter plot
 #' @import ggplot2
+#' @importFrom ggrastr geom_point_rast
```

Comment:
I think rather than importing this, which would require ggrastr -> `Imports:` in the description, I'd rather we just call the function by `ggrastr::geom_point_rast()` and then have an `.error_if_no_ggrastr()` within the first `if (do.raster)`.  See utils.R for similar error_if fxns.  And then please add to the DESCRIPTION in `Suggests:`